gnomad_mitochondria/manuscript_analyses/generate_Fig1A_4_3BCD_S4ABC_S5.R

```diff
@@ -0,0 +1,682 @@
+#!/usr/bin/env Rscript
+
+library(cowplot)
+library(dplyr)
+library(ggplot2)
+library(grid)
+library(optparse)
+library(tidyr)
+library(stringr)
+
+# Set up directories
+option_list = list(
+  make_option(c("-s", "--sample_annotations"), type = "character", default = NULL,
+              help = "The sample annotations file output by add_annotations.py for release samples", metavar = "character"),
+  make_option(c("-u", "--unfiltered_sample_annotations"), type = "character", default = NULL,
+              help = "Sample annotations for all, including filtered out samples, only needs to include mt_mean_coverage column", metavar = "character"),
+  make_option(c("-v", "--variant_data"), type = "character", default = NULL,
+              help = "Combined sites-only file (with txt extension) output by add_annotations.py", metavar = "character"),
+  make_option(c("-c", "--coverage"), type = "character", default = NULL,
+              help = "Mean coverages, tab-delimited file of 'locus', in format of chrM:position, and 'mean', mean coverage across all samples at that position (metrics can be obtained and extracted by running annotate_coverage.py)", metavar = "character"),
+  make_option(c("-d", "--plot_directory"), type = "character", default = NULL,
+              help = "Directory to which plots should be written", metavar = "character"))
+
+opt_parser <- OptionParser(option_list = option_list)
+opt <- parse_args(opt_parser)
+
+plot_dir <- opt$plot_directory 
+
+min_hom_threshold <- 0.95  # Minimum heteroplasmy level to define a variant as homoplasmic
+
+# Read in sample-level annotations generated by add_annotations.py
+data <- read.delim(opt$sample_annotations)
+data_unfiltered <- read.delim(opt$unfiltered_sample_annotations)
```

Comment:
Per above comment, if remove reference to this unfiltered file also remove this line